specifyweb/frontend/js_src/lib/components/Syncer/fromSimpleXmlNode.ts

```diff
@@ -5,6 +5,8 @@ import { formatXmlNode } from './formatXmlNode';
 import type { SimpleChildren, SimpleXmlNode, XmlNode } from './xmlToJson';
 import { getOriginalSyncerInput } from './xmlUtils';
 
+// Needed for RSS export
+export const fieldNeedsCamelCase = ['collectionId', 'userId', 'notifyUserId'];
```

Comment:
this is not good as it will affect all attributes called `collectionId` or `userId` in any syncer spec, not just the export feed spec - it would be very unexpected when some attributes across totally unrelated syncer files don't get converted to lowercase while others do
if you do want to solve this on the front-end, the solution has to be specific to the export feed spec
but I think this should be solved on the back-end